features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -50,6 +50,7 @@
   "oauth.token_validation.refresh_token_validity": "1d",
   "oauth.timestamp_skew": "0m",
   "oauth.token_renewal.renew_refresh_token": true,
+  "oauth.token_renewal.extend_refresh_token_expiry_time_on_renewal": false,
```

Comment:
shouldn't this be true?

Reply:
According to issue description, default behavior should be not extending the refresh token expiry time on token renewal. Since we are fixing this in the public shouldn't we disable this?

Reply:
Refresh token expiry time extension is enabled by default after having an offline discussion with @mefarazath, since according to the spec browser-based apps should only adhere to the spec.